lib/shopify_api/resources/order.rb

```diff
@@ -7,7 +7,7 @@ def close;  load_attributes_from_response(post(:close, {}, only_id)); end
     def open;   load_attributes_from_response(post(:open, {}, only_id)); end
 
     def cancel(options = {})
-      load_attributes_from_response(post(:cancel, options, only_id))
+      load_attributes_from_response(post(:cancel, {}, options.to_json))
```

Comment:
So in the request that was being submitted to Shopify it was an empty JSON object. The params were being encoded in the URL, which is weird for an action like this.

Running the code we just get `true` back, but that might be because of how the `load_attributes_from_response` behaves.